source/javascripts/homepage/_blog_posts.es6

```diff
@@ -0,0 +1,36 @@
+(function(window, $) {
+
+  function loadBlogPosts() {
+    const xhttp = new XMLHttpRequest();
+    xhttp.onreadystatechange = function() {
+      if (xhttp.readyState == 4 && xhttp.status == 200)
+        updatePosts(JSON.parse(xhttp.responseText));
+    };
+    xhttp.open("GET", "http://localhost:3000/blog/posts.json", true);
+    xhttp.send();
+
+    function updatePosts(posts) {
+      posts.splice(0, 2).forEach(post => {
+        const author = post.author;
+        const template = $('#blog-post-template').clone().find('.HorizontalGrid-cell');
+        template.find('.blog-post-title').html(post.title);
+        template.find('.blog-post-content').html(post.body);
```

Comment:
now that I see this, I realize you named the `processed_intro` attribute to `body` in the API.

I'm very much not a fan of that. there's already a `body` field in the post, which is not the one that's being sent, and can cause confusion

You mind if I change it back in the blog, which would require changing it here as well?